core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -722,35 +723,47 @@ public static String getDbVersion() {
     }
     
     public static String getDarwinAuthCheckUrl() {
-        String darwinAuthUrl = properties.getProperty(DARWIN_AUTH_URL);
-        if (darwinAuthUrl == null || darwinAuthUrl.isEmpty()) {
-            return "";
+        String darwinAuthUrl;
+        try{
+            darwinAuthUrl = properties.getProperty(DARWIN_AUTH_URL).trim();            
         }
-        return darwinAuthUrl.trim();
+        catch (NullPointerException e){
```

Comment:
Maybe simplify the code as below -- you can also try to catch any exceptions. Similar for other methods.

```
String darwinAuthUrl = "";
try {
    darwinAuthUrl = properties.getProperty(DARWIN_AUTH_URL).trim(); 
} catch (Exception e) {
}

return darwinAuthUrl;
```